### PR TITLE
Pocket extension read state filters

### PR DESCRIPTION
**Pull request by vimtor**
## Description

Add read state filters to the Pocket extension so users can search for `Read`, `Archived` and `All` types of bookmarks.

## Screencast

https://user-images.githubusercontent.com/36263538/159117538-79877939-2741-4019-a66e-16fda4f9922f.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `metadata` folder

### CODE REVIEW

File: extensions/pocket/changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Pocket Changelog
 
+## Read State Filters (2022-03-19)
+- Add `All`, `Unread` and `Archived` filters for the `Search Bookmarks` command
+- Add new preference to select default read state filter
+
 ## Categories & Screenshots (2022-03-12)
```
Comment (pernielsentikaer) [inline extensions/pocket/changelog.md:7]:
```suggestion
## [Categories & Screenshots] - 2022-03-12
```

File: extensions/pocket/changelog.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Pocket Changelog
 
+## Read State Filters (2022-03-19)
```
Comment (pernielsentikaer) [inline extensions/pocket/changelog.md:3]:
```suggestion
## [Read State Filters] - 2022-03-19
```

File: extensions/pocket/package.json [hunk 3]
```diff
@@ -63,7 +85,7 @@
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.27.1"
+    "@raycast/api": "^1.30.3"
```
Comment (pernielsentikaer) [inline extensions/pocket/package.json:88]:
```suggestion
    "@raycast/api": "^1.31.0"
```
If you want the newest 😄

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @vimtor 👋 

Thanks for the update, only left a few comments for you to address 🔥 

We released [1.31.0](https://developers.raycast.com/changelog#1.31.0-2022-03-23) with a lot of new awesome feature, be sure to check it out ⭐ 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by pernielsentikaer** [comment]
Hi @vimtor 👋 

Awesome update, thanks a lot!

Only one thing that I can imagine for now, that would make the UX more awesome 😄 

Would it be possible to add a [List.EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview) and make a pocket logo in 64x64px — but it can wait for the next version 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by vimtor** [comment]
Hey @pernielsentikaer I am not sure what you have in mind for the `List.EmptyView` do you want one different for each read state filter? Or a generic one that just says _No bookmarks found_?

In the meantime I did a few things:
- Added an `Open Latest Bookmark` command
- Renamed `Random Bookmark` to `Open Random Bookmark`
- Made `Open Random Bookmark` truly open a random bookmark
- Added a new command `Create Bookmark from Clipboard`

I was also considering adding a `Create Bookmark from Selection` but I wanted to ask you how to do it first. I've guess I could copy the current selection and then proceed with the `Create Bookmark from Clipboard` behaviour?

**Comment by pernielsentikaer** [comment]
Hi @vimtor 👋 

Thanks for the update, tested it, and it works quite nice! 

I was just thinking on a generic with an awesome pocket logo 😄 

I think that's a good idea to add, is it something you will add in this version?

**Pull request merged**